packages/server/src/express/middleware.ts

```diff
@@ -68,9 +83,18 @@ const factory = (options: MiddlewareOptions): Handler => {
                 zodSchemas,
                 logger: options.logger,
             });
-            response.status(r.status).json(marshalToObject(r.body, useSuperJson));
+            if (manageCustomResponse) {
```

Comment:
Just to make sure I understand how to use this. Since when `manageCustomResponse` is on the code here, simply returns the status and body, is it supposed to use the middleware like regular function call in a request handler then?

```js

const zenstack = ZenStackMiddleware({ manageCustomResponse: true, ... });

app.all('/api/model/*', async (req, res) => {
    const { status, body } = await zenstack(req, res);
    // more processing
    ...
    return res.status(status).json(body);
});

```